framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1598,15 +1600,15 @@ protected void analyze(
             boolean isStatic,
             @Nullable Store capturedStore) {
         ControlFlowGraph cfg = CFCFGBuilder.build(root, ast, checker, this, processingEnv);
-        /*
-             cfg.getAllNodes(this::isIgnoredExceptionType)
-                     .forEach(
-                             node -> {
-                                 if (node.getTree() != null) {
-                                     reachableNodes.add(node.getTree());
-                                 }
-                             });
-        */
+        if (checker.hasOption("notCheckDeadCode")) {
```

Comment:
Here, and probably in the `BaseTypeVisitor`, put the option into a protected variable, so that we don't look up the option multiple times. See how other options are handled.

Reply:
> Here, and probably in the `BaseTypeVisitor`, put the option into a protected variable, so that we don't look up the option multiple times. See how other options are handled.

addressed this comment in [2540cd3](https://github.com/eisop/checker-framework/pull/633/commits/2540cd301a1c598113e63b6555284a7e971228d8)